src/models/batch.py

```diff
@@ -26,7 +26,7 @@ def command(self):
 
     def __init__(self, **kwargs):
         self.config = kwargs['config']
-        self.arguments = ' '. join(kwargs['arguments'])
+        if 'arguments' in kwargs: self.arguments = ' '. join(kwargs['arguments'])
```

Comment:
I believe this has been added because `arguments` may not exist? It looks fine, but didn't see any mention of it.

Reply:
Ah yeah, apologies, I think it started breaking fairly abruptly so I threw this in. Think it was as a result of using batch run-family for a reason I didn't feel compelled to investigate.

Reply:
:+1: